nebula3/gclient/net/Connection.py

```diff
@@ -89,7 +93,9 @@ def open_SSL(self, ip, port, timeout, ssl_config=None):
             header_transport.open()
 
             self._connection = GraphService.Client(protocol)
-            resp = self._connection.verifyClientVersion(VerifyClientVersionReq())
+            verifyClientVersionReq = VerifyClientVersionReq()
+            verifyClientVersionReq.version = handshakeKey
```

Comment:
config the version for verifyClientVersionReq only when handshakeKey is not None.